nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -42,7 +42,7 @@ in {
 
     format = mkOption {
       type = types.enum [ "raw" "qcow2" "vpc" ];
-      default = "qcow2";
+      default = "vpc";
```

Comment:
Is `vpc` a sparse / compressed disk format? For Hydra it's important not to increase the size of the images a lot.

Reply:
I believe it's sparse but not compressed, comparisons:
```
└── [ 1193577472]  nixos-amazon-image-19.09pre130979.gfedcba-aarch64-linux.vhd
└── [ 1177550848]  nixos-amazon-image-19.09pre130979.gfedcba-aarch64-linux.qcow2
└── [ 1132745216]  nixos-amazon-image-19.09pre130979.gfedcba-x86_64-linux.vhd
└── [ 1116078080]  nixos-amazon-image-19.09pre130979.gfedcba-x86_64-linux.qcow2
```